reconcile/test/test_jenkins_worker_fleets.py

```diff
@@ -0,0 +1,45 @@
+from unittest import TestCase
```

Comment:
we have moved away from unittest. 

https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-testing-guide/#frameworks-and-tools